### DISCUSSION TITLE
Add Bit.ly provider

### DISCUSSION THREAD

**Discussion by Djiit** [nextauthjs/next-auth#3959] [Ideas]
### Description 📓

Hey team,

I'd like to be able to authenticate users with Bit.ly

Any plan about this provider?

Thanks!

### How to reproduce ☕️

N/A

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
If you want to give it a go, check out the docs https://next-auth.js.org/configuration/providers/oauth and the existing provider https://github.com/nextauthjs/next-auth/tree/main/packages/next-auth/src/providers

**Comment by Djiit** [comment]
Took a first stab at it. The doc is really helping to bootstrap the provider, well done. I'm stuck now, will need some help to debug it further.
